tensorflow_federated/python/core/impl/executors/federating_executor.py

```diff
@@ -35,6 +35,9 @@
 from tensorflow_federated.python.core.impl.executors import executor_utils
 from tensorflow_federated.python.core.impl.executors import executor_value_base
 
+import tf_encrypted as tfe
```

Comment:
you probably shouldn't pull in `tf_encrypted` yet -- it won't work on TF2. you can use `primitives` on its own.

Reply:
ah good point, I will remove this. thanks